src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -23,6 +23,15 @@ def session():
         yield simulated_session
 
 
+@pytest.fixture(scope='function')
+def session_5421():
+    with daqmx_sim_db_lock:
```

Comment:
The comment above "# We need a lock file so multiple tests aren't hitting the db at the same time" needs to be expanded and needs to reference the internal NI bug.

We should always reference a bug when we are working around a bug!

Specifically, it is not obvious why we grab a lock around init/close other than to a handful of NI driver engineers.

Reply:
I also think this should be own PR, it's not related to adding nitclk tests.